hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/util/PathAddressExtension.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.agent.monitor.util;
+
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.PathElement;
+
+/**
+ * PathAddress methods found on wildfly-core
+ * https://github.com/wildfly/wildfly-core/blob/588145a00ca5ed3beb0027f8f44be87db6689db3/controller/src/main/java/org/jboss/as/controller/PathAddress.java
+ */
+public class PathAddressExtension {
```

Comment:
Can you put all these compatibility static methods in the one WildflyCompabilityUtils.java and thus do not need to have a separate "OperationContextExtension" and "PathAddressExtension" class? This way, there is a single place for all the compatibility utilities. I don't see a need for these two extra "Extension" classes since they just have static methods.